README.md

```diff
@@ -71,13 +71,13 @@ To run the project , follow the instructions below .
           ./stop.sh
 
  ## Auto Regenerating the TX_PVLS Library resource    
- In case changes have been made to the Library cql logic at  `resources/cql_logic.txt`  , the  TX_PVLS Library resource  at  `resources/tx_pvls-library.json`  can be auto re-generated .
+ In case changes have been made to the Library cql logic at  `resources/tx_pvls_cql_logic.txt`  , the  TX_PVLS Library resource  at  `resources/tx_pvls-library.json`  can be auto re-generated .
```

Comment:
did you just forget adding the instructions for regenerating TX_CURR library resource ??